### PR TITLE
NIMBUS-16: Autocomplete feature

### PR DESCRIPTION
**Pull request by vivekkamineni2018**
# Description
Autocomplete feature.

The author of this pull request failed to include a description or DELETE this line.

# Overview of Changes

N/A

# Type of Change

N/A

# Test Details

N/A

# Additional Notes

N/A

### CODE REVIEW

**Comment by openanthem-admin** [comment]
You did it @vivekkamineni2018!

Thank you for signing the Anthem Opensource Software CLA.

File: nimbus-ui/nimbusui/src/app/components/platform/form/elements/autocomplete.component.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        </p-autoComplete>  
+  </ng-template>
+  
+   `
+})
+
+export class NmAutocomplete extends BaseControl<any> {
+
+    @ViewChild(NgModel) model: NgModel;
+    @Input() element: Param;
```
Comment (lopezton-anthem) [inline nimbus-ui/nimbusui/src/app/components/platform/form/elements/autocomplete.component.ts]: `@Input() element: Param;` is already declared in `BaseControl`.

File: nimbus-ui/nimbusui/src/app/components/platform/form/elements/autocomplete.component.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    @Input() form: FormGroup;
```
Comment (lopezton-anthem) [inline nimbus-ui/nimbusui/src/app/components/platform/form/elements/autocomplete.component.ts]: `@Input() form: FormGroup;` is already declared in `BaseControl`.

File: nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/cmd/exec/internal/DefaultCommandExecutorGateway.java [hunk 3]
```diff
@@ -216,10 +217,11 @@ protected MultiOutput executeInternal(ExecutionContext eCtx, CommandMessage cmdM
 		ExecutionConfig executionConfig = cmdParam != null ? cmdParam.getConfig().getExecutionConfig() : null;
 		
 		// if present, hand-off to each command within execution config
-		if(executionConfig != null && CollectionUtils.isNotEmpty(executionConfig.get())) {
-			List<MultiOutput> execConfigOutputs = executeConfig(eCtx, cmdParam, executionConfig.get());
-			execConfigOutputs.stream().forEach(mOut->addMultiOutput(mOutput, mOut));
-		}
+        //if(executionConfig != null && CollectionUtils.isNotEmpty(executionConfig.get())) {
```
Comment (lopezton-anthem) [inline nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/cmd/exec/internal/DefaultCommandExecutorGateway.java]: Remove commented code.

**Pull request merged**